api/v1alpha1/clusterinstance_types.go

```diff
@@ -89,6 +89,15 @@ const (
 	CPUPartitioningAllNodes CPUPartitioningMode = "AllNodes"
 )
 
+// CpuArchitecture is used to define the software architecture of a host.
+type CPUArchitecture string
+
+const (
+	// Supported architectures are x86 and arm
+	CPUArchitectureX86_64 CPUArchitecture = "x86_64"
+	CPUArchitectureAarch64 CPUArchitecture = "aarch64"
```

Comment:
I think we should a `multi` option to the list which can only be set at the ClusterInstanceSpec level. At a node level, we should allow either x86 or arm64: 
```go
CPUArchitectureMulti CPUArchitecture = "multi"
```